components/SideBar/SideBar.tsx

```diff
@@ -93,7 +81,7 @@ const SideBar: React.FC = () => {
         Connect to the UP extension to manage your portfolio
       </div>
       <button
-        onClick={loginExtension}
+        onClick={() => router.push('/connect-wallets')}
         className="flex items-center rounded-lg bg-deepPink py-1 px-4"
```

Comment:
I suggest to use Next Link instead: https://nextjs.org/docs/api-reference/next/link